include/opengemini/impl/cli/Query.cpp

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright 2024 Huawei Cloud Computing Technologies Co., Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "opengemini/impl/cli/Functor.hpp"
```

Comment:
this `#include` sort is unexpected.

Own header files (if any)
Standard library header files
Third-party library header files
Other header files of the current library

> We organize header files in this order, with a blank line between different types and alphabetical order within the same type.

Reply:
The `Functor.hpp` is the related header of `Query.cpp` so we put it first.